app/src/main/res/layout/sync_tabs_error_row.xml

```diff
@@ -24,12 +24,13 @@
 
     <com.google.android.material.button.MaterialButton
         android:id="@+id/sync_tabs_error_cta_button"
-        android:layout_width="wrap_content"
+        style="@style/PositiveButton"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
         android:maxLines="2"
-        style="@style/PositiveButton"
-        app:icon="@drawable/ic_sign_in"
-        android:visibility="gone"
         android:text="@string/synced_tabs_sign_in_button"
-        android:layout_marginTop="8dp"/>
```

Comment:
I think we should leave these in the same order, since the only thing changing here seems to be the `layout_width` 😄

Reply:
so should i change it again ?